crates/turborepo-auth/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 #![feature(cow_is_borrowed)]
 #![deny(clippy::all)]
+//! Turborepo's library for authenticating with the Vercel API.
+//! Handles logging into Vercel, verifying SSO, and storing the Vercel token.
```

Comment:
```suggestion
//! Handles logging into Vercel, verifying SSO, and storing the token.
```

Let's not say Vercel since it could be a custom remote cache